entrypoint.sh

```diff
@@ -228,6 +232,16 @@ function seiso_lint() {
 			linter["$key"]=$value
 		done < <(echo "$line" | jq -r 'to_entries|map("\(.key)=\(.value|tojson)")|.[]')
 
+		if [[ ${AUTO_FIX:-} == "true" ]]; then
+			if [[ -v linter[autofix] && -n "${linter[autofix]}" ]]; then
+				linter[args]="${linter[autofix]}"
```

Comment:
```suggestion
				# Replacing the linter's args with the autofix args for that linter
				linter[args]="${linter[autofix]}"
```

Reply:
Took me a minute of looking at this to figure out what was happening so suggest a comment